internal/clients/telegram/client.go

```diff
@@ -71,9 +75,12 @@ Creating the client is not enough, you have to `Start()` it.
 */
 func NewClient(host, token string, template template.Template) Client {
 	return Client{
-		host:     host,
-		basePath: "bot" + token,
-		client:   http.Client{},
+		requester: response.APIRequester{
+			Client:   http.Client{},
+			Scheme:   "https",
+			Host:     host,
+			BasePath: "bot" + token,
+		},
 		template: template,
```

Comment:
Instead of storing a template that is later converted by each thread that processes updates, store the ready to use `state.Responses`.